include/SFML/Window/WindowBase.hpp

```diff
@@ -180,35 +180,33 @@ class SFML_WINDOW_API WindowBase
     /// \brief Pop the next event from the front of the FIFO event queue, if any, and return it
     ///
     /// This function is not blocking: if there's no pending event then
-    /// it will return an empty event. Note that more than one event
+    /// it will return a `std::nullopt`. Note that more than one event
     /// may be present in the event queue, thus you should always call
     /// this function in a loop to make sure that you process every
     /// pending event.
     /// \code
-    /// while (const auto event = window.pollEvent())
+    /// while (const std::optional<sf::Event> event = window.pollEvent())
     /// {
     ///    // process event...
     /// }
     /// \endcode
     ///
-    /// \return The event; will be `Empty` (convertible to `false`) if no events are pending
+    /// \return The event; will be `std::nullopt` (convertible to `false`) if no events are pending
```

Comment:
```suggestion
    /// \return The event, otherwise `std::nullopt` if no events are pending
```
Tweaking the language to be more in line with how we document other `optional`-y APIs